pkg/R/utils.R

```diff
@@ -419,3 +419,82 @@ cleanClosure <- function(func, checkedFuncs = new.env()) {
   }
   func
 }
+
+# Common function for implmentation of zipRDD() and cartesian().
+# param
+#   x An RDD.
+#   other An RDD.
+#   operation The specified operation, "zip" or "cartesian".
+#
+# return value
+#   The result RDD.
+doZipOrCartesian <- function(x, other, operation = c("zip", "cartesian")) {
```

Comment:
For now, mergePartitions() needs only parameter. If we define two parameters for it: checkLengths , compactResults, they actually need to be mutually exclusive. If they are provided both true or false, undefined result will be got from the function. I don't know about any reuse of this function for now, so I think one parameter is enough. Until we find that this function can be reused, we can change the parameter definitions. What do you think?